packages/ui-core/src/components/slider/range-slider.component.tsx

```diff
@@ -144,15 +145,26 @@ export const RangeSlider: FC<Props> = ({
       };
 
   const [currentMinimum, currentMaximum] = value as number[];
+  const colorScale = generateContinousColorScale(
```

Comment:
As minimum, maximum, colorSteps are primitives types and additionally colors are array of strings - I suggest to use `useMemo` hook here to avoid scale creation for each range event change by the user.

Reply:
Good call 👍